### DISCUSSION TITLE
Will StrideEngine supprot gltf/glb 3d model format?

### DISCUSSION THREAD

**Discussion by DJDQfff** [stride3d/stride#1433] [Q&A] [answered]
Will stride support importing .gltf/.glb as 3d model ?
gltf has official csharp library gltfLoader and SharpGltf libraries.

**Comment by tebjan** [comment, marked as answer]
It already supports loading some gltf files. the ones that work with the assimp version that we currently use. also there are two pull requests related to that. you can ask specific questions there:
Assim update: https://github.com/stride3d/stride/pull/1158
gltf loader: https://github.com/stride3d/stride/pull/1030
